Makefile

```diff
@@ -27,7 +27,7 @@ test_lint:
 	}
 
 test_local:
-	go test -v -race -short -p=1 ./...
+	go test -v -short -p=1 ./...
```

Comment:
That's a dangerous change! You should be more concerned to change the tests so that they don't time-out. The `-race` saved us on numerous occasions, so if you decide to _really_ go without it, I'd make sure that the other members of the team are OK with it.

Reply:
So I removed the race because while I was updating onet i started having this error which only disappeared after removing the -race flag. I'm not sure where is this coming from :/
 
<img width="1095" alt="Screenshot 2020-03-11 at 10 04 45" src="https://user-images.githubusercontent.com/9702328/76399968-d6905100-637f-11ea-81ad-4c7dac712a4a.png">

Reply:
The plot thickens... apparently this only happens locally

Reply:
I think that is bbolt's error in conjunction with the latest go. Are you already using go 1.14?

@tharvik - what is the state of bbolt and the latest go?

Reply:
yes. If it's the case I can go back to a previous version

Reply:
@ineiti bbolt is still broken, we need https://github.com/etcd-io/bbolt/pull/201 before updating onet/cothority (but containerd and kubernetes both waits on that, it will be fixed in a matter of days)